features/step_definitions/et1_ccd_steps.rb

```diff
@@ -172,7 +175,7 @@
   ccd_object.assert_respondents(@respondent)
 
   expect(ccd_object.find_pdf_file).to match_et1_pdf_for(claim: @claim, claimants: @claimant, representative: @representative.first, respondents: @respondent, employment: @employment)
-  expect (ccd_object.as_json['response']['case_fields']['documentCollection'][1]['value']['uploadedDocument']['document_filename'].downcase) == (%W[et1_attachment_#{claimant[:first_name].underscore}_#{claimant[:last_name].downcase.delete("'")}.pdf])
+  expect(ccd_object.as_json['response']['case_fields']['documentCollection'][1]['value']['uploadedDocument']['document_filename'].downcase) == (%W[et1_attachment_#{claimant[:first_name].underscore}_#{claimant[:last_name].downcase.delete("'")}.pdf])
```

Comment:
This may just be me being pedantic, but we're expecting the first half of the == to equal the second half, not just expecting the first half, hence the spacing.

Reply:
Thinking about it could be even better to use .dig instead, so we can get this:
ccd_object.as_json.dig('response', 'case_fields', 'documentCollection', 1, 'value', 'uploadedDocument', 'document_filename').to_s.downcase == (%W[et1_attachment_#{claimant[:first_name].underscore}_#{claimant[:last_name].downcase.delete("'")}.pdf])